### DISCUSSION TITLE
Suggestion: Add night tree as background

### DISCUSSION THREAD

**Discussion by hosttry** [BuddiesOfBudgie/budgie-backgrounds#19] [Background Suggestions]
Description
----
Night sky view with stars and some trees.

Image Information
-
- Author:  Free Nature Stock https://stocksnap.io/author/freenaturestock
- License:
StockSnap's CC0 License https://stocksnap.io/license
Creative Commons CC0 license https://creativecommons.org/publicdomain/zero/1.0/
- Source: https://stocksnap.io/photo/night-tree-DSOUXZLKJ8

Aditional
-
I like this picture but i don´t know if it´s too busy.

**Comment by serebit** [comment]
Cursory glance, looks like a valid submission to me (correct license, resolution, content).

Cropped, scaled, and set as background:
![image](https://user-images.githubusercontent.com/12981608/194336606-0d833f6d-0b7e-4cf0-8ac4-3361a9b80caf.png)
I like this a lot, though the fact that the trees on the left side are out of focus does bother me a bit. @JoshStrobl thoughts?
